app/models/mixins/relationship_mixin.rb

```diff
@@ -527,10 +581,14 @@ def child_types(*args)
   end
 
   def add_parent(parent)
+    return parent.add_children(self) if use_ancestry?
+
     parent.with_relationship_type(relationship_type) { parent.add_child(self) }
   end
 
   def add_children(*child_objs)
+    return child_objs.each { |child| child.with_relationship_type(relationship_type) { child.update!(:parent => self) } } if use_ancestry?
+
```

Comment:
so after this call, the list of my children will not be updated.

wonder if the caller of `add_children` expects this or if they are not looking at this object's children after this call

Reply:
well regardless, either the child will have the wrong parent or the parent will have the wrong children. This is the nature of active record in general.  I'm ok with this line.